lib/asar/fs-wrapper.ts

```diff
@@ -30,11 +30,13 @@ const getOrCreateArchive = (archivePath: string) => {
     return cachedArchives.get(archivePath);
   }
 
-  const newArchive = asar.createArchive(archivePath);
-  if (!newArchive) return null;
-
-  cachedArchives.set(archivePath, newArchive);
-  return newArchive;
+  try {
+    const newArchive = new asar.Archive(archivePath);
+    cachedArchives.set(archivePath, newArchive);
+    return newArchive;
+  } catch (_) {
```

Comment:
```suggestion
  } catch {
```
removing the `_` should be fine if we want to swallow the error. It is already done in places like https://github.com/electron/electron/blob/e589e9b259ecad5a2f2f47f0b8d40618d1c54601/lib/browser/api/app.ts#L77.